concepts/Vulnerability_disclosure_and_embargo_policy/00004-Vulnerability_disclosure_and_embargo_policy.md

```diff
@@ -0,0 +1,221 @@
+# RFC00004: Vulnerability Disclosure and Embargo Policy
+- Authors: [axel simon](github@axelsimon.net)
+- Status: [PROPOSED](/README.md#proposed)
+- Since: 2020-03-10
+- Status Note: under discussion
+- Supersedes: N/A
+- Start Date: 2020-03-03 (date you started working on this idea)
+- Tags: security, infrastructure
+
+## Summary
+
+Describe the security vulnerability disclosure process, including embargoes.
+
+## Motivation
+
+Users and researchers who discover security issues in Enarx need to know that
+we welcome their efforts and how we will manage their disclosure. They must
+also be able to have an idea of how we use security embargoes.
+
+In other words:
+- We want:
+  - to encourage reporting of security issues (ie: “attack us!”)
+  - to make it easy to report security issues
+  - to maintain and extend our policy of safe defaults / discouraging people
+  from adopting bad security practices
+  - to recognise and show appreciation for the time spent researching and
+  reporting
+  - to make it possible for people who may be at risk to report issues
+  pseudonymously
+
+- We need:
+  - a clear policy, describing both:
+    - how to report; and
+    - what to expect during the process
+  - a main reporting channel:
+    - secondary channels, should the main one fail/not be available/be
+  overloaded.
+  - a list of people who can be reached directly
+  - a place to list security reports and give attribution
+
+
+## Tutorial
+
+This RFC aims to provide a set of documents (wiki pages) that describe the 
```

Comment:
Maybe specify that these will *turn into* wiki pages, with a link to the wiki?

Reply:
I've linked to https://github.com/enarx/enarx/wiki/Vulnerability-disclosure-and-embargo-policy below even though that doesn't exist yet, but in this specific case it's more of a general explanation of what the RFC is about, so i'm not sure a link is necessary. Does that make sense?
Also, as i'm not 100% sure the documents will be wiki pages (although i think they will), i worded it that way.

Reply:
```suggestion
This RFC aims to provide a set of documents (ex. wiki pages) that describe the 
```